### DISCUSSION TITLE
Can't use image augumentation in iamge prediction.

### DISCUSSION THREAD

**Discussion by xinlnix** [autogluon/autogluon#1812] [Q&A]
My original data is in jepg format.
When I using the autogluon, I can't apply my image argument method, like transforms in pytorch.

**Comment by zhiqiangdon** [comment]
Our `MultiModalPredictor` can train image classifiers now. See the tutorial: https://auto.gluon.ai/stable/tutorials/multimodal/beginner_image_cls.html

In `MultiModalPredictor`, you can specify image augmentations like this:

```
# default used by MultiModalPredictor
predictor.fit(
hyperparameters={
"model.timm_image.train_transform_types": ["resize_shorter_side", "center_crop", "trivial_augment"],
}
) # We transform the images sequentially with the operations given in the list

# you can specify to use RandAugment
predictor.fit(
hyperparameters={
"model.timm_image.train_transform_types": ["resize_shorter_side", "center_crop", "randaug"],
}
)

```
TrivialAugment: https://arxiv.org/abs/2103.10158
RandAugment: https://arxiv.org/abs/1909.13719

You can also use Mixup in image augmentations. See the tutorial here: 
https://auto.gluon.ai/stable/tutorials/multimodal/customization.html#data-mixup-turn-on
